app/assets/icons/index.ts

```diff
@@ -0,0 +1,38 @@
+import messages from "./message-square-02.png"
+import notification from "./bell-01.png"
+import evidencForm from "./Frame.png"
+import saved from "./bookmark.png"
+import support from "./annotation-question.png"
+import profile from "./user-01.png"
+import search from "./search-md.png"
+import eye from "./eye.png"
+import brush from "./brush-01.svg"
+import dot from "./dot.png"
+import users from "./users-01.png"
+import music from "./music-note-01.svg"
+import pencil from "./pencil-line.svg"
+import star from "./star-06.svg"
+import plus from "./plus.png"
+import home from "./home.svg"
+import drop from "./chevron-down.png"
```

Comment:
Please use intuitive names for the icons. A good way to improve readability and maintainability is to use names that convey the meaning.
For eg: 
**messageIcon** instead of messages.
**notificationIcon** instead of notification.

This clearly indicates that the import is an Icon.
Also, you can rename the images as well user-01, brush-01 etc to user, brush.